packages/plugin-ext/src/plugin/tests.ts

```diff
@@ -483,9 +491,18 @@ export class TestRunProfile implements theia.TestRunProfile {
     @observableProperty('notifyPropertyChange')
     label: string;
 
-    @observableProperty('notifyPropertyChange')
+    @observableProperty('notifyIsDefaultChange')
     isDefault: boolean;
 
+    private onDidChangeDefaultEmitter = new Emitter<boolean>();
+    onDidChangeDefault = this.onDidChangeDefaultEmitter.event;
+
+    protected notifyIsDefaultChange(_property: keyof TestRunProfileDTO, isDefault: boolean): void {
```

Comment:
I suspect this will generate an extra call to the front end:

1. Front set "isDefault"
2. TestingExt.$onChangeDefault() is called
3. notifyDefaultChanged is invoked and sends `$updateTestRunProfile()`

I don't think the "observableProperty" mechanism works for this: we need to distinguish where a change is coming from.

Reply:
I updated to avoid this circular call. Only one call shall be made now.